fe/src/components/Navbar/NavbarProfile.vue

```diff
@@ -54,6 +54,9 @@ export default {
     },
     viewProfile() {
       // Implement action to view profile
+      this.$router.push('/ClientViewProfile')
+
```

Comment:
remove extra spaces

Reply:
I removed the extra space already po ma'am.